seb_openedx/edxapp_wrapper/backends/get_chapter_from_location_h_v1.py

```diff
@@ -0,0 +1,16 @@
+""" Backend abstraction """
+from opaque_keys.edx.keys import UsageKey
+from openedx.core.lib.url_utils import unquote_slashes  # pylint: disable=import-error
+from xmodule.modulestore.django import modulestore  # pylint: disable=import-error
+from xmodule.search import path_to_location  # pylint: disable=import-error
+
+
+def get_chapter_from_location(usage_id, course_key):
+    """ hawthorn backend """
+    usage_key = UsageKey.from_string(unquote_slashes(usage_id)).map_into_course(course_key)
+    if usage_key:
+        path = path_to_location(modulestore(), usage_key)
+        chapter_index = 1
```

Comment:
Would you like making this a CHAPTER_INDEX "constant" ?

Reply:
Ok